### PR TITLE
demumble 1.2.2 (new formula)

### PR DESCRIPTION
**Pull request by jlaxson**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/demumble.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  sha256 "13f89330abbae24262d1c7a0717c25e71737b727e72db78ff77d8ca35566c22f"
+  license "Apache-2.0"
+  head "https://github.com/nico/demumble.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    bin.install "build/demumble"
```
Comment (carlocab) [inline Formula/demumble.rb:14]:
```suggestion
    system "cmake", "--install", "build"
```
Does this not work here?

Comment (jlaxson) [reply Formula/demumble.rb:14]: No, upstream has no CPack/install rules in the cmake file

File: Formula/demumble.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class Demumble < Formula
+  desc "More powerful symbol demangler (a la c++filt)"
+  homepage "https://github.com/nico/demumble"
+  url "https://github.com/nico/demumble/archive/refs/tags/v1.2.2.tar.gz"
+  sha256 "13f89330abbae24262d1c7a0717c25e71737b727e72db78ff77d8ca35566c22f"
```
Comment (carlocab) [inline Formula/demumble.rb:5]: You have a checksum mismatch:
```
Error: SHA256 mismatch
Expected: 13f89330abbae24262d1c7a0717c25e71737b727e72db78ff77d8ca35566c22f
  Actual: 663e5d205c83cc36a257bb168d3ecbc2a49693088c0451b2405d25646651c63e
```
You probably got this from piping `curl` to `sha256sum`, or similar:
```
❯ curl -s https://github.com/nico/demumble/archive/refs/tags/v1.2.2.tar.gz | sha256sum
13f89330abbae24262d1c7a0717c25e71737b727e72db78ff77d8ca35566c22f  -
```
But this just gives you the checksum of GitHub's redirection stub:
```
❯ curl -s https://github.com/nico/demumble/archive/refs/tags/v1.2.2.tar.gz
<html><body>You are being <a href="https://codeload.github.com/nico/demumble/tar.gz/refs/tags/v1.2.2">redirected</a>.</body></html>
```
Instead, what you want is
```
❯ curl -Ls https://github.com/nico/demumble/archive/refs/tags/v1.2.2.tar.gz | sha256sum
663e5d205c83cc36a257bb168d3ecbc2a49693088c0451b2405d25646651c63e  -
```

Comment (jlaxson) [reply Formula/demumble.rb]: Guilty as charged... And I thought I watched for the correct size download.

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**